TeamCharles/initial-site
# 92-viewAllProductsBtn - All Products Button

## Status

**READY**
## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[X] This PR does **NOT** require documentation.
## Description

A button on the Home Page that redirects the user to a list of all products
## Resolves Issue Number

https://github.com/TeamCharles/initial-site/issues/92
## Deploy Notes

``` sh
git pull --prune
git checkout 92-viewAllProductsBtn
dotnet run
```
## Steps to Test or Reproduce
1.  Go to `localhost:5000`
2. Click `View All Products` button
3. You should be redirected to `localhost:5000/Products`
## Impacted Files in Application
- Views/Products/Index.cshtml
